components/link/link.scss

```diff
@@ -14,12 +14,17 @@
 		color: $d2l-color-celestine-minus-1;
 		text-decoration: underline;
 	}
-	&:focus,
-	&.d2l-link-focus {
+	&:focus-visible {
 		outline: 2px solid $d2l-color-celestine;
 		outline-offset: 1px;
 		text-decoration: underline;
 	}
+	.d2l-focus-visible-not-supported &:focus {
```

Comment:
Yeah, this would be added in BSI. I have the change ready, just didn't put up a PR since I wasn't sure how palatable this change would be. It would import the helper and just do:
```javascript
if (!isFocusVisibleSupported()) {
	document.documentElement.classList.add('d2l-focus-visible-not-supported');
}

```